ui/apps/platform/src/Containers/Dashboard/PatternFly/SummaryCounts.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import Raven from 'raven-js';
+import pluralize from 'pluralize';
+import { useQuery } from '@apollo/client';
+import { Spinner, Split, Stack } from '@patternfly/react-core';
+
+import { SummaryCountsResponse, SUMMARY_COUNTS } from 'queries/summaryCounts';
+
+type SummaryTileProps = {
+    label: string;
+    value: number;
+    isLoading: boolean;
+    isError: boolean;
+};
+
+/*
+  TODO
+  We have a `TileContent` component that was used in the old version of this
+  widget at ./apps/platform/src/Components/TileContent/TileContent.tsx
```

Comment:
The widget referenced here is used in a handful of pages that are on the old style. e.g.) Old main dashboard SummaryCounts, VulnMgmt -> Deployments -> Sidebar

I wasn't sure if this is a pattern we want to retain in PF, or if it is going to be slowly replaced as we migrate old pages. I created this Tile as a one-off for the summaries instead of porting the old Tile but can do that instead if it is something we need to use going forward.